src/lib/CMakeLists.txt

```diff
@@ -379,7 +385,7 @@ if (NOT DISABLE_JIT_SUPPORT)
         ${SOURCES}
         ${JIT_EMBEDDED_SOURCES}
         ${LLVM_BUNDLE_FILE}
-    )
+        storage/mvcc_columns.cpp storage/mvcc_columns.hpp)
```

Comment:
this seems to be a duplicate of lines 267–268

Reply:
here is CLion trying to be smarter than I am